man/systemd.net-naming-scheme.xml

```diff
@@ -531,6 +531,20 @@
     particular version of systemd).</para>
   </refsect1>
 
+  <refsect1>
+    <title>Limit reading specific sysfs attributes</title>
+
+    <para>When creating names for network cards, some naming schemes use data from sysfs populated
+    by the kernel. This means that although a specific naming scheme in udev is picked,
+    the network card's name can still change when a new kernel version adds a new sysfs attribute.
+    Users can set an allowlist or denylist for these. To create an allowlist, the user needs to set
+    <varname>ID_NET_NAME_ALLOW=0</varname> for the device and then list the allowed attributes in the format
+    of <varname>ID_NET_NAME_SYSFSATTR=1</varname>. In case of a deny list, the user needs to set
```

Comment:
`ID_NET_NAME_SYSFSATTR=1` -> `ID_NET_NAME_ALLOW_SYSFSATTR=1`

Reply:
Also, this should probably be `ID_NET_NAME_<replaceable>SYSFSATTR</replaceable>=1` .

Reply:
fixed